vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -3182,9 +3182,15 @@ private void manageBannedHeader(RoomMember member) {
                     VectorUtils.getRoomDisplayName(this, mSession, mRoom), mRoom.getState().getMemberName(member.mSender)));
         }
 
+        // On mobile side, the modal to allow to add a reason to ban/kick someone isn't yet implemented
+        // That's why, we don't display the TextView "Motif :" for now.
         TextView subInvitationTextView = findViewById(R.id.room_preview_subinvitation_textview);
         subInvitationTextView.setText(getString(R.string.reason_colon, member.reason));
-
+        if (null != member.reason) {
+            subInvitationTextView.setText(getString(R.string.reason_colon, member.reason));
```

Comment:
Make it 2 lines : 
`final String reason = getString(R.string.reason_colon, member.reason);
  subInvitationTextView.setText(reason)`